Krati Agrawal/Loops/hcf.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+using namespace std;
+int main(){
+    int num1, num2, min, hcf=1;
+    cout<<"Enter first number= ";
+    cin>>num1;
+    cout << "Enter second number= ";
+    cin>>num2;
+    min=(num1<num2)? num1: num2;
+    for(int i=1;i<=min;i++){
+        hcf=i;
```

Comment:
you have miss the if condition

Reply:
plz check for line 9

Reply:
`if (num1 % i == 0 && num2 % i ==0) {
      hcf = i;
  }`

It should be this

Reply:
Thanks for pointing out. I've corrected the code , plz check.